doc/src/quickstart/index.rst

```diff
@@ -485,6 +391,135 @@ which we can visualize with:
         --route_chan_width 100 \
         --analysis --disp on
 
+Manually Running VTR with ODIN II
+----------------------------------
+VTR includes a second synthesis tool, ODIN II. Below we explain how to run this alternative synthesis flow.
```

Comment:
I would say "we explain how to manually run". Just to be clear that this is not the recommended way, but its possible if you want to avoid the scripts.